pkg/ffmpeg/codec_hardware.go

```diff
@@ -310,6 +332,9 @@ func (f *FFMpeg) hwApplyFullHWFilter(args VideoFilter, codec VideoCodec, fullhw
 		if fullhw && f.version.Gteq(Version{major: 3, minor: 3}) { // Added in FFMpeg 3.3
 			args = args.Append("scale_qsv=format=nv12")
 		}
+	case VideoCodecRK264:
+		// For Rockchip, no extra mapping here. If there is no scale filter,
+		// leave frames in DRM_PRIME for the encoder.
```

Comment:
Does the Rockchip decoder change for the format to nv12 so we dont need to enforce it here for full hardware encoding?